src/server/routes/__tests__/submissions.integration.spec.js

```diff
@@ -381,4 +382,196 @@ describe('submissions.integration', () => {
       expect(result.payload).toBe('')
     })
   })
+
+  describe('GET /api/submissions/{submissionId}/activites', () => {
+    const CONTACT_IDENTIFIER_GET_ACTIVITIES_FOR_SUBMISSION =
+      'contact-identifier-get-activities-for-submission'
+
+    beforeEach(() =>
+      deleteActivitiesAndSubmissions(
+        CONTACT_IDENTIFIER_GET_ACTIVITIES_FOR_SUBMISSION
+      )
+    )
+
+    afterAll(() =>
+      deleteActivitiesAndSubmissions(
+        CONTACT_IDENTIFIER_GET_ACTIVITIES_FOR_SUBMISSION
+      )
+    )
+    it('should successfully get all activivities for a submission with a valid submission id', async () => {
+      // create a submission
+      const createdSubmission = await server.inject({
+        method: 'POST',
+        url: '/api/submissions',
+        payload: {
+          contactId: CONTACT_IDENTIFIER_GET_ACTIVITIES_FOR_SUBMISSION,
+          season: '2023',
+          status: 'INCOMPLETE',
+          source: 'WEB'
+        }
+      })
+
+      const submissionId = JSON.parse(createdSubmission.payload).id
+
+      // add 2 activities with different rivers
+      const createdActivity1 = await server.inject({
+        method: 'POST',
+        url: '/api/activities',
+        payload: {
+          submission: `submissions/${submissionId}`,
+          daysFishedWithMandatoryRelease: '20',
+          daysFishedOther: '10',
+          river: 'rivers/3'
+        }
+      })
+      const createdActivity1Id = JSON.parse(createdActivity1.payload).id
+
+      const createdActivity2 = await server.inject({
+        method: 'POST',
+        url: '/api/activities',
+        payload: {
+          submission: `submissions/${submissionId}`,
+          daysFishedWithMandatoryRelease: '5',
+          daysFishedOther: '3',
+          river: 'rivers/1'
+        }
+      })
+      const createdActivity2Id = JSON.parse(createdActivity2.payload).id
+
+      const result = await server.inject({
+        method: 'GET',
+        url: `/api/submissions/${submissionId}/activities`,
+        payload: {
+          submission: `submissions/${submissionId}`,
+          daysFishedWithMandatoryRelease: '5',
+          daysFishedOther: '3',
+          river: 'rivers/1'
+        }
```

Comment:
These injected payloads are all remarkably similar - I think there could be scope for a generation function to create it?